src/Picqer/Carriers/SendCloud/Connection.php

```diff
@@ -112,12 +112,16 @@ public function apiUrl()
      * Perform a GET request
      * @param string $url
      * @param array $params
+     * @param string $auth
      * @return array
      * @throws SendCloudApiException
      */
-    public function get($url, $params = [])
+    public function get($url, $params = [], $auth = false)
     {
         try {
+            if ($auth) {
+                $params[$auth] = $this->apiKey;
```

Comment:
Nope, to leave all possibilities open, the $auth param could be filled with whatever string you want. 
For example to fetch the service points the auth parameter is filled in with 'access_token':
$result = $this->connection->get($url, $params, 'access_token');